integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/TaskFhirResourceIntegrationTest.java

```diff
@@ -169,6 +176,45 @@ public void shouldCreateNewTaskAsJson() throws Exception {
 		assertThat(newTask.getIntent(), equalTo(task.getIntent()));
 		assertThat(task.getAuthoredOn(), equalTo(task.getAuthoredOn()));
 		assertThat(task.getLastModified(), equalTo(task.getLastModified()));
+		assertThat(newTask.getOutput(), hasSize(4));
+		assertThat(newTask.getInput(), hasSize(4));
+		for (Task.TaskOutputComponent output : newTask.getOutput()) {
```

Comment:
Why write the verifications like this? The order is semi-important, right, or at least we should be able to predict what it is here, so we can just do:

```java
assertThat(newTask.getOutput().get(0), /* ... */)
```

Which is much more straight-forward to understand than these loops.